fastlane/lib/fastlane/actions/app_store_connect_api_key.rb

```diff
@@ -2,10 +2,6 @@
 
 module Fastlane
   module Actions
-    module SharedValues
-      APP_STORE_CONNECT_API_KEY = :APP_STORE_CONNECT_API_KEY
```

Comment:
do we really want to break API for users who might depend on this?

Reply:
I was unsure about this. I didn’t super want to keep it but I also didn’t want to break 😬 I can put it back in!